bin/zerobufCxx.py

```diff
@@ -15,6 +15,8 @@
 import sys
 from collections import namedtuple, OrderedDict
 
+inline_implementation = False
```

Comment:
Please check https://github.com/HBPVIS/ZeroBuf/commit/7ab0058e24cc262eb95a27a344b4a6bee78429cf; there is already enough code to decide if to use inline on Windows, and not-inline everywhere else. I wonder why I forgot the inline back then, it seemed to have worked, but that's obviously not working.